README.md

```diff
@@ -1,24 +1,113 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h1>Hello React Back End</h1>
 
-* Ruby version
+</div>
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🗝️ Key Features](#key-features)
+  - [🛠️ Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database creation
+# 📖 Hello React Front End <a name="about-project"></a>
```

Comment:
- [ ] @AyomipoMajek Great work implementing the changes requested however you forgot to change the title in line 25 as previously requested, kindly consider doing it
![image](https://github.com/AyomipoMajek/hello-rails-backend/assets/61408860/b535fc23-fbae-4f09-9bce-57eddd3eb7e8)